src/components/network/network-map-tab.tsx

```diff
@@ -1100,7 +1071,13 @@ export const NetworkMapTab = ({
                     height: '100%',
                     '& .mapbox-gl-draw_polygon': {
                         // To override the bg-color of the draw button when we enter in draw mode
-                        backgroundColor: isInDrawingMode ? 'lightblue !important' : 'inherit',
+                        backgroundColor: isInDrawingMode.value ? 'lightblue !important' : 'inherit',
```

Comment:
We may use a theme color, instead of this 'lightblue' not depending of the mode light/dark